client/src/pages/locations/Form.js

```diff
@@ -444,6 +468,21 @@ const LocationForm = ({
       location
     })
   }
+
+  async function checkPotentialDuplicates(locationName) {
+    if (!edit && locationName.length >= MIN_CHARS_FOR_DUPLICATES) {
+      const queryParams = Object.assign({
+        text: locationName
```

Comment:
If you only want the total count, you should set `pageSize: 1` on the query for optimum performance.